source/math/FloatRandomGenerator.ooc

```diff
@@ -31,22 +32,18 @@ FloatUniformRandomGenerator: class extends FloatRandomGenerator {
 	_min, _max, _rangeCoefficient: Float
 	minimum ::= this _min
 	maximum ::= this _max
-	init: func {
-		this _state = Time microtime()
-		this setRange(0.0f, 1.0f)
-	}
-	init: func ~withSeed (seed: Int) {
-		this _state = seed
+	init: func (seed := Time microtime()) {
+		this _state = FloatRandomGenerator permanentSeed != 0 ? This permanentSeed : seed
```

Comment:
`this _state = This permanentSeed != 0 ? This permanentSeed : seed` doesn't work?